# 磁铁位姿信息
std_msgs/Header header
geometry_msgs/Point position
geometry_msgs/Quaternion orientation
float64 magnetic_strength
